go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/googleapis/gax-go
+
+require google.golang.org/grpc v1.16.0
```

Comment:
Shouldn't this depend on `v2`? Could you run `go mod tidy`?

Reply:
I had thought it would need to, also. I have run `go mod tidy` (see: `go.sum`). I'm investigating this oddity and several related oddities (see: [1], [2], [3], [4], [5]). Will update once I find out.

1: https://github.com/golang/go/issues/28685
2: https://github.com/golang/go/issues/27056
3: https://github.com/golang/go/issues/26241
4: https://gist.github.com/myitcv/79c3f12372e13b0cbbdf0411c8c46fd5
5: (will add once the googlegroup link becomes available - sitting in golang-nuts moderation queue)